Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+  REPORTS: "舉報",
+  WELCOMES: "歡迎訊息",
+  WELCOMES_VARIABLES: "歡迎訊息變數",
+  WELCOME_ROLE: "預設身分組",
+  CLOUDFLARE: "Cloudflare",
+  COMMON_ERRORS: "常見錯誤"
+};
+
+exports.GENERAL = {
+  NOT_AVAILABLE: "不可用的",
+  STEP: "步驟 {{number}}",
+  EXAMPLE: "範例",
+  AND: "和",
+  DO_NOT_INCLUDE_THIS: "當您在打指令時，請勿包含 ``<>``",
+  CHECK_DISCORD_SUPPORT_ARTICLE: "點**[這裡]({{link}})**來查看Discord官方的說明",
+  OPTIONAL: "可填",
+  PERMANENT: "永久",
+  TEMPORAL: "暫時",
+  TEMPORARY: "暫時",
+  PERMANENTLY: "永久",
+  CHANNEL: "頻道l",
+  CHANNEL_ID: "頻道 ID",
+  ROLE: "身分組",
+  ROLE_ID: "身分組 ID",
+  THRESHOLD: "進入點",
+  TIME: "時間",
+  IF_APPLICABLE: "如果適用",
+  LEVEL: "等級",
+  NUMBER: "數字",
+  ACTION: "動作",
+  COMMAND: "指令",
+  MESSAGE: "訊息",
+  USER: "使用者",
+  SERVER: "伺服器",
+  NAME: "名稱"
+};
+
+exports.ABOUT_WIKI = {
+  TITLE: "歡迎來到 Filo 指南百科",
+  FILO_DESCRIPTION: "Filo 是個多功能且強大的 Discord 機器人。 高自訂性、多語言、檢舉系統、日誌、歡迎和離別訊息、人數統計、反邀請、AUTO MODERATION、反規避，以及 +190 個指令",
+  WIKI_DESCRIPTION: " Filo 指南百科的內容會不斷改變以提高其品質 . 本指南涵蓋了一些常見錯誤，以及每個 Filo 指令的指令教學 ",
+  ASK_FOR_HELP: "如果本指南沒有解決您想要的問題，歡迎拜訪我們的 **[Discord 伺服器]({{link}})**",
+  WIKI_KEYS: {
+    TITLE: "指南小提示",
+    DESCRIPTION: "在每個網頁會有一個小提示，它們可以讓您考慮一些重要的資訊。 您可以在下面範例找到這裡提到的一些小提示",
+    SUCCESS_KEY: "說明或建議事項將會顯示在與此類小提示類似的框中",
+    INFO_KEY: "信息或註解將會顯示在與此類小提示類似的框中",
+    WARNING_KEY: "警告將會顯示在與此類小提示類似的框中",
+    DANGER_KEY: "重要訊息將會顯示在與此類小提示類似的框中"
+  },
+  REQUEST_TRANSLATION: {
+    TITLE: "協助翻譯或修正內容！",
+    DESCRIPTION: "如果您需要，您可以要求 更新 或 修改 這份指南的任何主題。無論是為了改進還是為了更正過時或已棄用的用詞" + "\n" + "這份指南擁有著許多語言，都是志願者提供的幫助。如果您想要協助我們翻譯，**[請點擊這裡]({{link}})**"
+  }
+};
+
+exports.HELPFUL_LINKS = {
+  TITLE: "實用連結"
+};
+
+exports.NAVBAR = {
+  HOME: "指南首頁",
+  DISCORD_SERVER: "Discord 伺服器",
+  PRIVACY_POLICY: "隱私政策",
+  TERMS_AND_CONDITIONS: "條款及細則",
+  WEBSITE: "官方網站",
+  DONATE: "捐款",
+  BLOG: "部落格",
+  TWITTER: "Twitter",
+  VOTE: "投票",
+  GETTING_STARTED: "入門",
+  INVITE_FILO: "邀請 Filo",
+  MODULES: "模組",
+  LANGUAGE: "語言",
+  PREFIX: "Prefix",
+  GUILD: "伺服器",
+  REPORTS: "檢舉",
+  WELCOMES: "歡迎",
+  WELCOME_ROLE: "預設身分組",
+  FAREWELLS: "離別訊息",
+  MEMBER_COUNTER: "人數統計",
+  MEMBER_COUNTER_NICKNAME: "人數統計 (暱稱)",
+  LOGGING: "日誌",
+  ANTI_INVITES: "反邀請",
+  AUTO_MODERATION: "AUTO MODERATION",
+  ANTI_EVASION: "反規避",
+  ANTI_SPAM: "反洗版",
+  ANTI_MASS_MENTIONS: "防止提及",
+  ANTI_REPEATED_TEXT: "返重複文字",
+  DISABLE_COMMANDS: "停用指令",
+  ANTI_BAD_WORDS: "反壞話",
+  COMMANDS: "指令",
+  TOPICS: "主題",
+  CLOUDFLARE: "Cloudflare",
+  COMMON_ERRORS: "常見錯誤"
+};
+
+exports.HOME = {
+  NAVIGATION_INDEX: {
+    TITLE: "導覽目錄",
+    GETTING_STARTED: {
+      DESCRIPTION: "如果你剛加入 Filo 的行列，你可以從下面的連結查看必要資訊"
+    }
+  },
+  MODULES: {
+    DESCRIPTION: "在這裡您可以找到各種關於模組的重要資訊"
+  }
+};
+
+exports.GETTING_STARTED = {
+  INVITE_FILO: {
+    DESCRIPTION: "接下來你會看到一個伺服器邀請的例子，還有我們建議的權限設定",
+    PERMISSIONS_NEEDED: "你必須是伺服器 **擁有者** 或是擁有這些權限： {{permission_1}} 或 {{permission_2}} 才能將 Filo 邀請至伺服器",
+    SLASH_COMMAND_WARNING: "目前 Filo 將不支援沒有把機器人斜線指令權限開啟的伺服器，或是註冊 Filo 斜線指令但把 Filo 踢出的伺服器，如此一來你將會收到一個一般錯誤",
```

Comment:
I do not know if it is an error in the English translation, but this line is absolutely wrong.

The correct translation is: "Currently, Filo does not support the use of slash commands without the scope bot in the invite. If you add slash command integration to your server and don't add Filo to the server as well, slash commands will always return a generic English error"